### PR TITLE
add tests with race detector

### PR DESCRIPTION
**Pull request by lwsanty**
Signed-off-by: lwsanty <lwsanty@gmail.com>

---

This PR contains changes that do not require a mention in the CHANGELOG file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@ jobs:
         # only linux supports docker https://docs.travis-ci.com/user/docker/ => tests with docker skipped programmatically ~ OS
         - docker-compose up -d
         - make test-coverage codecov
+        - make test-race
```
Comment (dpordomingo) [inline .travis.yml]: why using `make ci-script` instead?
https://github.com/src-d/ci/blob/v1/Makefile.main#L278

Comment (lwsanty) [reply .travis.yml]: @dpordomingo I forgot that my PR to `ci` was merged already xD thank you!

Comment (lwsanty) [reply .travis.yml]: Done

Comment (dpordomingo) [reply .travis.yml]: thanks for PR this!

**Pull request merged**